modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -13,16 +13,22 @@ This guide walks you through querying Redpanda topics as Iceberg tables in https
 == Prerequisites
 
 * xref:manage:tiered-storage.adoc#configure-object-storage[Object storage configured] for your cluster and xref:manage:tiered-storage.adoc#enable-tiered-storage[Tiered Storage enabled] for the topics for which you want to generate Iceberg tables.
-** The S3 bucket URI so that you can configure it as external storage for Open Catalog.
++
+You need the S3 bucket URI, so you can configure it as external storage for Open Catalog.
 * A Snowflake account.
-* An Open Catalog account. To https://other-docs.snowflake.com/en/opencatalog/create-open-catalog-account[create an Open Catalog account], you require ORGADMIN access in Snowflake.
+* An Open Catalog account. To https://other-docs.snowflake.com/en/opencatalog/create-open-catalog-account[create an Open Catalog account^], you require ORGADMIN access in Snowflake.
 * An internal catalog created in Open Catalog with your Tiered Storage AWS S3 bucket configured as external storage.
```

Comment:
@kbatuigas 
I know the steps in this page are already long, but the reality is that the catalog,  volume creation and IAM policy creation are really part of the setup and not prerequisites, because they all relate only to this particular Redpanda' cluster's tiered storage and exposing it to Snowflake (e.g. I think the catalog is the 'redpanda' catalog).  The true prerequisites are are the things above this line only.

So, we should include step by step instructions for catalog , volume and IAM policy creation here and not call them prerequisites.

Reply:
Created new [issue](https://redpandadata.atlassian.net/browse/DOC-1086)